src/gui/gui.h

```diff
@@ -69,10 +70,12 @@ class DissolveWindow : public QMainWindow
     QLabel *restartFileIndicator_;
     // Label for iteration number
     QLabel *iterationLabel_;
-    // Label for simulation ETA (when using RunFor)
+    // Label for simulation timer (ETA if RunFor, time elapsed if Run)
     QLabel *etaLabel_;
```

Comment:
I think this warrants a name change, given the new purpose of the label.
```suggestion
    QLabel *timerLabel_;
```